packages/contracts-bedrock/test/kontrol/proofs/utils/DeploymentSummary.sol

```diff
@@ -0,0 +1,269 @@
+pragma solidity ^0.8.13;
```

Comment:
The autogenerated files lack `// SPDX-License-Identifier: <license>` statements which will result in a compiler warnings. Forge suppresses this one by default, but we probably should default to MIT here for consistency, since it's autogenerated code